src/transformers/openformat.js

```diff
@@ -0,0 +1,44 @@
+var parseString = require('xml2js').parseString;
+var stripNS = require('xml2js').processors.stripPrefix;
+
+async function callOpenformat(params, context) {
+  const url = context.get('services.openformat', true);
+
+  // Add fields from request
+  const fields = {};
+  params.fields.map(field => (fields[field] = `{${field}}`));
+
+  const data = [];
+  for (var i = 0; i < params.pids.length; i++) {
+    const xmlResult = await context.request(url, {
```

Comment:
Denne function bliver lidt kompleks, så jeg ville tage det meste inde i for-loopet og pakke det ind i en async funktion som returnerer data. Og så bør du pakke funktionen ind i en try catch som kan fange og logge eventuelle fejl.